framework/utilities.cpp

```diff
@@ -211,6 +211,35 @@ bool isEnemy(D2::Types::UnitAny* unit) {
     return unitHP(unit) > 0 && isHostile(unit) && isAttackable(unit);
 }
 
+std::vector<D2::Types::UnitAny*> getUnits(int type) {
```

Comment:
needs to accept the hash table as an argument (in case we're doing it server side with pGame instead of client side units)